platform/default/src/mbgl/layermanager/layer_manager.cpp

```diff
@@ -69,6 +73,12 @@ LayerManagerDefault::LayerManagerDefault() {
     addLayerType(std::make_unique<LocationIndicatorLayerFactory>());
 #endif
 #endif
+
+#if MLN_DRAWABLE_RENDERER
+#if !defined(MLN_LAYER_CUSTOM_DRAWABLE_DISABLE_ALL)
```

Comment:
This is the only place where this define is used.

Should we include it by default for now, and do a pass later to allow people to not include this functionality?

Reply:
I followed the pattern used for the other layer types. it can be done either way, let's discuss about it.

Reply:
Discussed!